meson.build

```diff
@@ -49,6 +50,7 @@ screentest_srcs = [
 executable(
   meson.project_name(),
   'main.c',
+  c_args: '-DGSEAL_ENABLE -DGTK_DISABLE_DEPRECATED -DGTK_DISABLE_SINGLE_INCLUDES',
```

Comment:
When switching to GTK+ 3 I'd drop these again. They only serve to highlight the progress of the migration.